test/extended/olm/olm.go

```diff
@@ -240,25 +240,25 @@ var _ = g.Describe("[sig-operator] an end user can use OLM", func() {
 		oc = exutil.NewCLI("olm-23440")
 
 		buildPruningBaseDir = exutil.FixturePath("testdata", "olm")
-		operatorGroup       = filepath.Join(buildPruningBaseDir, "operatorgroup.yaml")
-		sub                 = filepath.Join(buildPruningBaseDir, "subscription.yaml")
+		ogPath              = filepath.Join(buildPruningBaseDir, "operatorgroup.yaml")
+		subPath             = filepath.Join(buildPruningBaseDir, "subscription.yaml")
+		catalogPath         = filepath.Join(buildPruningBaseDir, "catalogsource.yaml")
 	)
 
-	files := []string{sub}
 	g.It("can subscribe to the operator", func() {
 		g.By("Cluster-admin user subscribe the operator resource")
 
-		// skip test if redhat-operators is not present or disabled
-		ok, err := hasRedHatOperatorsSource(oc)
+		// Configure OperatorGroup before tests
+		og, err := oc.AsAdmin().Run("process").Args("--ignore-unknown-parameters=true", "-f", ogPath, "-p", "NAME=test-operator", fmt.Sprintf("NAMESPACE=%s", oc.Namespace())).OutputToFile("og.json")
+		o.Expect(err).NotTo(o.HaveOccurred())
+		err = oc.AsAdmin().WithoutNamespace().Run("create").Args("-f", og).Execute()
 		o.Expect(err).NotTo(o.HaveOccurred())
-		if !ok {
-			g.Skip("redhat-operators source not found in enabled sources")
-		}
 
-		// configure OperatorGroup before tests
-		configFile, err := oc.AsAdmin().Run("process").Args("--ignore-unknown-parameters=true", "-f", operatorGroup, "-p", "NAME=test-operator", fmt.Sprintf("NAMESPACE=%s", oc.Namespace())).OutputToFile("config.json")
+		// Configure CatalogSource before tests
+		const image = "registry.redhat.io/redhat/redhat-operator-index:v4.10"
```

Comment:
this is going to be a source of bit-rot someday.

can we identify the clusterversion and pick the version tag based on that, dynamically?

Reply:
or pull the value in from elsewhere?  (i know we also have the challenge that the day we start v4.12, there may not be a v4.12 catalog published, so that could be a problem too...  maybe n-1 is the tag to use)

Reply:
yeah i guess maybe pinning to 4.10 and if someday that breaks on a v4.16 cluster, we can bump it at that point.  not ideal, but not worth holding this up over.